apps/dashboard/app/models/configuration_property.rb

```diff
@@ -0,0 +1,61 @@
+class ConfigurationProperty
```

Comment:
This file is in `models` the other in `apps`.  Can we move them both to models? (you can use `git mv` to preserve the git diff).

Reply:
ok - will do